### PR TITLE
Adding gegelati library package

### PR DESCRIPTION
**Pull request by lucascarvalhoroncoroni**
This pull request is a contribution to add a new package to Spack.
Gegelati is a TPG library, a lightweight reinforcement learning algorithm implemented in C++.
I though would be good to have an AI library among all Spack packages.

### CODE REVIEW

File: var/spack/repos/builtin/packages/gegelati/package.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+from spack.package import *
+
+
+class Gegelati(CMakePackage):
+    """Class to add the a TPG library(gegelati) into Spack"""
+
+    homepage = "https://github.com/gegelati/gegelati"
+    url = "https://github.com/gegelati/gegelati/archive/refs/tags/v1.2.0.tar.gz"
+
+    # notify when the package is updated.
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/gegelati/package.py:15]: This comment is no longer necessary.  Suggest removing it *unless* you really want to keep it.

**Review by tldahlgren — APPROVED** [review]
Confirmed the sha256.  Let me know if you want this PR merged with the maintainers comment in place.

**Comment by tldahlgren** [comment]
FYI. I removed reference to "resource" in the title since it could be confused with resource definitions in Spack (see https://spack.readthedocs.io/en/latest/packaging_guide.html#resources-expanding-extra-tarballs).

**Comment by lucascarvalhoroncoroni** [comment]
Thank you

Em qui., 3 de nov. de 2022 15:00, Tamara Dahlgren ***@***.***>
escreveu:

File: var/spack/repos/builtin/packages/gegelati/package.py (hunk 1 above)
Comment (lucascarvalhoroncoroni) [reply var/spack/repos/builtin/packages/gegelati/package.py:15]: No worries. I want to keep it.
Thank you :)

**Pull request merged**